recipes/globus-compute-endpoint/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "globus-compute-endpoint" %}
+{% set version = "2.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/globus-compute-endpoint-{{ version }}.tar.gz
+  sha256: 212d1a43a6ca7318f1826d23620749b813e8be94c9b636679912e3cd426fd377
+
+build:
+  skip: true  # [not unix]
```

Comment:
This is a platform dependent noarch python package. I think the skip statement is warranted.